test/README.md

```diff
@@ -0,0 +1,95 @@
+## How To Test Verdaccio
+
+Welcome to the testing folder at Verdaccio. This document aims to help you to go throug how Verdaccio show be tested.
+
+First of all we must explain the stack, we use `jest` and other tools as `supertest` to be able test the API and `puppetter` for End to End testing.
+
+We go along with the following rules in order to be consisten with all test and will make your code review smooth and fast:
+
+* We **type** all our test. eg `const foo: number = 3;`
+* **Each test should be small as possible**: You should try the `test()` block try to test only one thing and no dependencies with other test. If test requires different steps, groupd them woth `describe()` block.
```

Comment:
small as => as small as
block try => block
no dependencies with other test => do not depend on other tests
If test => If the test
groupd => group
woth => with a